src/DADynamoApp/SetupDARequest.cs

```diff
@@ -10,7 +57,19 @@ internal class SetupDARequest
         //
         // Summary:
         // Save the revit document to the default result.rvt file.
-        [JsonProperty("saveRvt", Required = Required.DisallowNull, NullValueHandling = NullValueHandling.Ignore)]
-        public bool SaveRvt { get; set; } = false;
```

Comment:
is this class something you have designed or is it something inherited by us from DA?